lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -151,7 +152,7 @@ type ListTransactions t = "wallets"
 
 -- | https://input-output-hk.github.io/cardano-wallet/api/#operation/postExternalTransaction
 type PostExternalTransaction = "external-transactions"
-    :> ReqBody '[JSON] PostExternalTransactionData
+    :> ReqBody '[JSON, OctetStream] PostExternalTransactionData
```

Comment:
What is the rationale/benefit of having support for both types, couldn't we just have either `Json` or `octet-stream`? (just asking because it basically means more testing)

Reply:
> What is the rationale/benefit of having support for both types, couldn't we just have either `Json` or `octet-stream`? (just asking because it basically means more testing)

If there's no clear requirement to support both, then I would also be in favour of cutting this down to just one.

Reply:
I was thinking about it and initially I was persuaded to choose one. Now, I am inclined to support both. Json is standard in our api, all users (on all technical levels) know how to use it. But on the other hand, more technical users will probably favor `octet-stream`. Frankly speaking when testing, I started to choose this content type. So for the time being,  I propose to leave it as it is (both content types reuse >95% of code), and decide to nuke one of them after talking with product representative.

Reply:
I don't think it's wise to support both here. JSON is a bit awkward as we are really just sending plain bytes, so `OctetStream` is more suitable. If any of the two has to prevail, I'd strongly suggest to keep `OctetStream`.